cmd/create/externalauthproviders/cmd.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externalauthproviders
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/externalauthproviders"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var externalAuthProvidersArgs *externalauthproviders.ExternalAuthProvidersArgs
+
+const argsPrefix string = ""
+
+var Cmd = &cobra.Command{
+	Use:     "external-auth-provider",
+	Aliases: []string{"externalauthproviders", "externalauthprovider", "external-auth-providers"},
+	Short:   "Create an external authentication provider for a cluster.",
+	Long:    "Configuring the cluster with external authentication configuration instead of internal oidc provider. ",
```

Comment:
This long description is not clear to me, maybe too many "configuring" and "configuration"

Maybe:
"Configure a cluster to use an external authentication provider instead of an internal oidc provider."

Reply:
Changed texts